### PR TITLE
zh-CN: 'Diagnostics' translations

### PR DESCRIPTION
**Pull request by ghost**
Refs: https://github.com/nodejs/nodejs.org/pull/4728

### CODE REVIEW

**Comment by MaledongGit** [comment]
@nodejs/nodejs-cn ：Plz have a review of these translations, notice this series of translations tries my best to fit the Chinese's thoughts instead of hard translations for most to understand.

**Comment by ghost** [comment]
@willin  ：Plz have a review of these translations, notice this series of translations tries my best to fit the Chinese's thoughts instead of hard translations for most to understand.

File: locale/zh-cn/docs/guides/diagnostics/memory/using-heap-profiler.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+对堆剖析器的采样是在一定时间内持续跟踪内存份分配状况，以及后备内存。由于采样基于低
+开销进行，所以它可以用在生产环境。
+
+> 你可以借助 [`heap-profiler`][] 模块，通过编程方式实现此功能。
+
+### 如何对堆剖析进行采样？
+
+启动应用程序：
+
+```console
```
Comment (willin) [inline locale/zh-cn/docs/guides/diagnostics/memory/using-heap-profiler.md:76]: is it `bash`?

File: locale/zh-cn/docs/guides/diagnostics/memory/using-heap-snapshot.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+![采集内存快照][take a heap snapshot image]
+
+#### 2. 使用 `--heapsnapshot-signal` 命令符
+
+> 此方法在 Node.js v12.0.0 或之后版本使用
+
+你可以使用命令行的命令符启动 node 使得对信号做出反应并开始创建内存快照。
+
+```
```
Comment (willin) [inline locale/zh-cn/docs/guides/diagnostics/memory/using-heap-snapshot.md:46]: does not have a type here

**Review by willin — APPROVED** [review]
LGTM

File: locale/zh-cn/docs/guides/diagnostics/memory/using-heap-profiler.md (hunk 1 above)
Comment (ghost) [reply locale/zh-cn/docs/guides/diagnostics/memory/using-heap-profiler.md:76]: I'm afraid not, it's something like in Linux as a command line. And it's directly copied from the English version.
For bash, it looks like something here:

File: locale/zh-cn/docs/guides/diagnostics/memory/using-heap-snapshot.md (hunk 2 above)
Comment (ghost) [reply locale/zh-cn/docs/guides/diagnostics/memory/using-heap-snapshot.md:46]: Yes, there're also many English versions with it without a fatal issue. So I temporarily decide not to change it and I'll change it at 2nd version. I found something changed in another version, I'll submit another changed version for these issues. Thank you.

**Pull request merged**